upp-pub-provisioner/ansible/userdata/initial_user_data.yaml

```diff
@@ -12,6 +12,7 @@ write_files:
       carousel_bucket={{carousel_bucket}}
       carousel_enabled={{carousel_enabled}}
       clusterid={{clusterid}}
+      cms_notifier_credentials={{cms_notifier_credentials}}
```

Comment:
It already exists in the vault - it just wasn't being passed through to the final cloud config that was being applied.